jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/sam/SamExecutable.kt

```diff
@@ -309,3 +313,12 @@ fun GeneralCommandLine.samSyncCommand(
         addParameter("--code")
     }
 }
+
+fun addSourceAsToolkitInEnvVariable(environmentVariables: Map<String, String>): Map<String, String> {
```

Comment:
should be placed closer to executable resolution so we don't need to remember to add this env var every time